blog/_posts/2017-08-10-opensource-here-we-come.md

```diff
@@ -0,0 +1,90 @@
+
+## The Crew
+
+### Coaches
+![image](/img/blog/2017/teamgemini_coaches.jpg)
+<div align="center" class="image-credits">Our coaches(Photo: Facebook)</div>
+
+#### [Akshay Agarwal](https://www.facebook.com/akshay.agarwal.1675275) 
+
+He is a doctoral student working in the field of biometrics and machine learning at IIITD. He has completed his Mtech in from IIIT Allahabad. He recently went to West Virginia University for a 9-month Internship program. He also works in other fields like Pattern recognition, Face recognition, face spoofing, Image analysis to name a few. 
+
+#### [Shiven Mian](https://github.com/shivenmian/)
+
+He is a junior at IIITD, majoring in Computer science, Polygot Programmer, passionate about Opensource, with a side of research. Shiven is a 2016 GSoC intern with Fossasia and worked in Loklak.
+
+
+#### [Aditay Tripathi](https://github.com/Aditay/)
+
+He has just completed his MTech from our college only and did his BTech from NIT Shrinagar. 
+He is an AI enthusiast in Research and applications of Machine learning/deep learning in speech recognition and computer vision. He is well versed with various libraries like Kaldi, Tensorflow, Theano and Keras.
+
+### Mentors
+![image](/img/blog/2017/teamgemini_mentors.jpg)
+<div align="center" class="image-credits"> Our mentors(Photo: twitter)</div>
+
+#### [Hong phuc dang](https://github.com/hpdang)
+
+Hpdang is internationally educated business professional with focus on open technologies, developer community outreach and business development. She has extensive knowledge and technical expertise in information technologies, Open Source applications and developer community engagement.
+
+#### [Mario Behling](https://github.com/mariobehling)
+
+Mario is the co-founder of Fossasia and the founder of Susi.AI. He works as the Development lead and Fossasia summit organiser. He is also the founder of a startup named awaike. 
+
+### Supervisor 
+#### [Vaishali](https://github.com/nerdyvaishali)
+
+She is an Open Source enthusiast and works as a Linux kernel engineer at Oracle. She is also a blogger and outreachy alumni. She motivates everyone to get started with linux kernel and also takes up workshops for the same.
+
+## What we’ve done so far and the Road ahead
+
+Time indeed flies. We’ve already sifted through the large code base, understanding it, made small PRs here and there, learnt git, understand what a frontend architecture like react does and realise the effort that a community does in maintaining a project. It was one hectic month! We are really proud of the progress we’ve made in our open source journey so far.
```

Comment:
If you want to go for past tense then do that for all phrases here ["understand -> understood" ].
Also, change "realise the effort that a community does ..." -> " **Realized** **an** effort community **puts** ..."